### ISSUE TITLE
Could not find accessory in dictionary

### ISSUE THREAD

**Issue by OLTSYS** [basdelfos/homebridge-tuya-web#24]
Hello,

Configuring Tuya-web for homebridge has been a successfuly operation ; I can control through Homekit one light bulb, two basic switchs and one dimmer. Everything is running fine 👍 

However, I got a cycle error message displayed by the homebridge console : 
"[2019-8-30 3:28:19 AM] [TuyaWebPlatform] Could not find accessory in dictionary"

Can't get the origin of this message. 

More detailed logs with the -D ebug option : 
```
[2019-8-30 3:33:34 AM] [TuyaWebPlatform] Refreshing state for all devices...
[2019-8-30 3:33:35 AM] [TuyaWebPlatform] [UPDATING][Ampoule Color Mezzanine]: { brightness: '255',
  color_mode: 'colour',
  color: { saturation: 100, brightness: 100, hue: 316 },
  online: true,
  state: 'true' }
[2019-8-30 3:32:14 AM] [TuyaWebPlatform] [UPDATING][Lampe Color Mezzanine]: { online: true, state: true }
[2019-8-30 3:32:14 AM] [TuyaWebPlatform] [UPDATING][VMC Chambre]: { brightness: '106', online: true, state: 'true' }
[2019-8-30 3:32:14 AM] [TuyaWebPlatform] [UPDATING][Lampe Bureau]: { online: true, state: false }
[2019-8-30 3:32:14 AM] [TuyaWebPlatform] Could not find accessory in dictionary
[2019-8-30 3:32:14 AM] [TuyaWebPlatform] Could not find accessory in dictionary
[2019-8-30 3:32:14 AM] [TuyaWebPlatform] Could not find accessory in dictionary
[2019-8-30 3:32:14 AM] [TuyaWebPlatform] Could not find accessory in dictionary
```
I have deleted the _Persist_ folder and _Accessories_ folder from the _.homebridge_ directory, with no more success.

Could you guess where it comes from ?
Thanks

**Comment by basdelfos** [comment]
Looks like you have some devices that are registered in the App that are not discovered by the plugin on initial loading of the plugin.
When the plugin periodically refreshed the state from the backend API there are 4 devices reported back that are not available (not discovered) by the plugin, the plugin then logs this error message.
Could be that not all devices are supported by the backend API and therefor are not registered in the plugin, do you have more Tuya devices in the App than are registered in Homebridge/Home App?

**Comment by OLTSYS** [comment]
Thank you for your reply.

Sadly, there is no more devices in my SmartLife Tuya app than the 4 detected by the homebridge plugin ; 4 devices in the App, 4 devices detected and available in Homekit.
However I should have found the origin of those errors ; it seems to be linked to the Scene & Automation Rules in the SmartLife App. I have 2 Scenes and 2 Automations ... for a total of 4 error messages. I tried to add a new Scene, and I got... a fifth error message in Homebridge. 

Solution should be to delete Scenes and Automations from the SmartLife App, especially since everything can be generated from the Scene of the Homekit App now :) 

Thank you again for your help.
